packages/compass-aggregations/src/modules/input-documents.ts

```diff
@@ -100,13 +97,6 @@ export const refreshInputDocuments = (): PipelineBuilderThunkAction<
       return;
     }
 
-    if (dataService && !dataService.isConnected()) {
-      debug(
-        'warning: trying to refresh aggregation but dataService is disconnected',
-        dataService
-      );
-    }
```

Comment:
This was adding a dependency on a never before used data service method in this plugin just to print a debug statement, feels like a bit too much, so I removed it